WebRoot/js/zimbraAdmin/accounts/controller/ZaAccountViewController.js

```diff
@@ -705,3 +709,17 @@ function(elementValue) {
 	}
 	return false;	
 }
+
+ZaAccountViewController.prototype._getLDAPAttr = 
+function(email) {
+	if (email) {
+		let attrString = 'uid=';
+		let emailStringArray = email.split('@');
+		attrString = attrString.concat(emailStringArray[0]);
+		attrString = attrString.concat(',ou=people');
```

Comment:
I observed usage of two concat in single string.
Can't we use plus (+) operator?

Reply:
Fixed.